ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -277,6 +277,12 @@ then
 			pwm disarmed -c 6 -p p:PWM_AUX_DIS6 -d ${OUTPUT_AUX_DEV}
 			pwm disarmed -c 7 -p p:PWM_AUX_DIS7 -d ${OUTPUT_AUX_DEV}
 			pwm disarmed -c 8 -p p:PWM_AUX_DIS8 -d ${OUTPUT_AUX_DEV}
+			pwm disarmed -m 0x100 -p p:PWM_MAIN_DIS9 -d ${OUTPUT_AUX_DEV}
```

Comment:
What's going on here?

Reply:
You mean the `-m`? Because `-c 10` would select channels 1 and 0.

Reply:
Ha, seems like a good reminder we should find a new way to do this.